### ISSUE TITLE
Move pet image in the staff pets show pages

### ISSUE THREAD

**Issue by mononoken** [rubyforgood/pet-rescue#862]
## Description

**tl;dr:** We want to remove the right side image from the staff pets show page and add a thumbnail in its place to the header of the page.

We have received negative feedback from developers in regards to the image found on the staff pets show page. Currently, this page has multiple tabs for staff to navigate through for a pet. There is an image of the pet to the right.

Many of these tabs could be reusing partials from other pages. However, those partials in other locations do not have this image, and so we end up having to redesign a partial to be used for multiple pages but with different width constrictions. The image does not serve any particular benefit either other than helping signal to staff which pet they are looking at currently.

We want to remove the image, add a thumbnail, and we also need to adjust some of the html/css here. With the image removed, we will want the content of the page to take up the space that the image used to occupy. That way, we can share partials from other pages and they will have the same space constraints.

We also may need to adjust the html to allow for the thumbnail to exist as shown in the screenshot.

And finally, make sure the layout still works well for mobile.

![Image](https://github.com/rubyforgood/pet-rescue/assets/81536479/18d893ae-702e-4a54-9afb-b196170c747d)

https://www.figma.com/design/0jVgYASUJy0KiX3BVc3dFM/Tasks?node-id=1094-70&t=WZjsRcM8EoT4DZdu-1
_Note: Ignore the content in the box; this PR is only concerned about the thumbnail_
![Image](https://github.com/rubyforgood/pet-rescue/assets/81536479/3652789c-4a73-4341-b0bf-840eb928e827)

## Acceptance Criteria
- [ ] Remove the image to the right side
- [ ] Add thumbnail image to the header space of the page layout per figma design
- [ ] Page content extends full page width in lieu of removed image

**Comment by Aaryanpal** [comment]
Hi @mononoken I can take up this issue. Can you please assign it to me? Thanks

**Comment by jmilljr24** [comment]
@Aaryanpal All yours! Let us know if you have any questions.

**Comment by kasugaijin** [comment]
How's this doing @Aaryanpal ?

**Comment by kasugaijin** [comment]
bump @Aaryanpal need any help on this?

**Comment by kasugaijin** [comment]
@Aaryanpal thanks for the update! No rush at all. Take your time. I just like to make sure people are still interested otherwise I like to reassign.

**Comment by Aaryanpal** [comment]
@kasugaijin Thanks for Understanding.

**Comment by Aaryanpal** [comment]
@kasugaijin 
Is it Seems Right? the image size

![Image_check_2](https://github.com/user-attachments/assets/8e237d44-7a8b-4773-adf5-04244373d02f)
